README.md

```diff
@@ -39,25 +35,22 @@
 <br/>
 <br/>
 
-
 <!-- ABOUT THE PROJECT -->
 # About The Project
 
-thin-edge.io is the first open-source and cloud-agnostic edge framework designed for resource-constrained IoT edge devices.  
+thin-edge.io is the first open-source and cloud-agnostic edge framework designed for resource-constrained IoT edge devices.
 
-We have started the project to solve the challenge of IoT device enablement in industrial IoT or more precisely to make the integration of IoT devices with cloud and IoT platforms easy and reliable.  
+We have started the project to solve the challenge of IoT device enablement in industrial IoT or more precisely to make the integration of IoT devices with cloud and IoT platforms easy and reliable.
```

Comment:
Two NBSPs between
- `make` and `the`
- `and` and  `reliable`
Some editor make the NBSPs visible. From GH, we can see the NBSP looks narrower gap than the normal spaces.